tests/testthat/test_calc_ss_analytic.R

```diff
@@ -94,6 +94,8 @@ test_that("2-cmt infusion", {
 })
 
 test_that("3-cmt oral", {
+  skip_on_cran()
+  pk_3cmt_oral <- new_ode_model("pk_3cmt_oral")
```

Comment:
we are building this model (and the iv version) twice now, which makes our testing a little slower when CRAN is set to false. Can we bring it out of the test_that chunks and wrap it in the `if (identical(Sys.getenv("NOT_CRAN"), "true")) { ...}` check that you used elsewhere?